lib/hako/container.rb

```diff
@@ -87,6 +87,20 @@ def log_configuration
       end
     end
 
+    # @return[Hash, nil]
+    def health_check
+      if @definition.key?('health_check')
+        conf = @definition['health_check']
+        {
+          command: conf.fetch('command'),
+          interval: conf.fetch('interval'),
```

Comment:
interval, timeout, retries, and start_period seems optional according to the documentation.

Reply:
thanks, I fixed.